src/status_im/subs.cljs

```diff
@@ -722,9 +723,10 @@
  :<- [:contacts/contacts]
  :<- [:multiaccount]
  (fn [[contacts multiaccount] [_ id]]
-   (or (:photo-path (contacts id))
-       (when (= id (:public-key multiaccount))
-         (:photo-path multiaccount)))))
+   (multiaccounts/displayed-photo (or (contacts id)
+                                      (when (= id (:public-key multiaccount))
```

Comment:
this condition will return a photo-path so it will always recompute the photo because displayed-photo expects a map